packages/mroonga-package-task.rb

```diff
@@ -84,6 +84,17 @@ def detect_mysql_version_debian(code_name)
     end
   end
 
+  def ubuntu_package_name(package_name)
+    ubuntu_package_names = {
+      "mariadb-10.2" => "mariadb-server-10.2",
+      "mariadb-10.3" => "mariadb-server-10.3",
+      "mariadb-10.4" => "mariadb-server-10.4",
+      "mariadb-10.5" => "mariadb-server-10.5",
+      "mysql-5.7" => "mysql-server-5.7",
+    }
```

Comment:
It seems that we can generate Ubuntu package name from our package name: `package_name.sub(/-/, "-server")`

We don't want to maintain this mapping. If MariaDB or MySQL release a new series, we need to update this mapping.

Reply:
I see.
I'll fix it.